OpenRA.Mods.yupgi_alert/OpenRA.Mods.yupgi_alert.csproj

```diff
@@ -2,9 +2,7 @@
 <Project DefaultTargets="Build" ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
-    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
-    <ProductVersion>10.0.0</ProductVersion>
-    <SchemaVersion>2.0</SchemaVersion>
+    <Platform Condition=" '$(Platform)' == '' ">x86</Platform>
```

Comment:
The mod dlls do work as AnyCPU, but the engine can't, due to dependencies from outside .NETverse like SDL where we're clearly using x86. Also https://github.com/OpenRA/OpenRA/issues/12587#issuecomment-272854670